src/app/dashboard-widgets/create-work-item-widget/create-work-item-widget.component.ts

```diff
@@ -1,8 +1,8 @@
 import { Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
 import { FilterService, WorkItem, WorkItemService } from 'fabric8-planner';
-import { Contexts } from 'ngx-fabric8-wit';
-import { UserService } from 'ngx-login-client';
-import { ConnectableObservable, Observable } from 'rxjs';
+import { Contexts, Space } from 'ngx-fabric8-wit';
+import { User, UserService } from 'ngx-login-client';
```

Comment:
UserService isn't used anymore, and there are currently a few other unused imports that can be cleaned up (e.g., ConnectableObservable, publishReplay, switchMap)